lib/puppet_metadata/aio.rb

```diff
@@ -15,8 +15,8 @@ class AIO
         '5' => 5..7,
         '6' => 5..7,
         '7' => 5..7,
```

Comment:
Looks like there are also EL7 builds @ https://yum.puppet.com/puppet8/el/index.html
```suggestion
        '7' => 5..8,
```